next.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = withTM({
 		tsconfigPath: './tsconfig.build.json',
 	},
 	images: {
-		domains: ['assets.viaa.be'],
+		domains: ['assets.viaa.be', 'via.placeholder.com'],
```

Comment:
Is die `via.placeholder.com` nog nodig of was deze enkel voor te testen?

Reply:
In principe enkel om te testen. Blijft er idealiter in tot dat we thumbnails kunnen renderen. ik zet er een TODO bij.